include/fem-dem/gls_vans.h

```diff
@@ -148,9 +148,65 @@ class GLSVANSSolver : public GLSNavierStokesSolver<dim>
   void
   read_dem();
 
+
+  /**
+   * @brief This function gives the periodic offset of the domain which is needed
+   * for the periodic boundary conditions using the QCM for void fraction.
+   *
+   * @param periodic_boundary_id The id of the periodic boundary 0
```

Comment:
You mean 0 is the default pbc? If so, it would be clearer if it was said "... boundary. Boundary 0 is used by default", or smth like this.

Reply:
It's not really a default value, this is just the one that is chosen between 0 and 1, it would work with 1 also.

Reply:
Actually, it's quite tricky because there's PBC in DEM and CFD. The GLS VANS has no idea about anything related to DEM, so it uses periodic for CFD which are "id" and "periodic_id" and not "periodic boundary 0" and "periodic boundary 1" as with DEM. I used the same name as CFD for DEM at first, but the "id" what used for outlet and other boundaries and it was confusing. I will just remove any mention of pbc 0 in this part.